File: JAVASCRIPT/Cap04/c04ex02.js
// c04ex02.js

const rls = require('readline-sync');

let A = rls.questionInt('Entre o 1o. valor numerico: ');
let B = rls.questionInt('Entre o 2o. valor numerico: ');

if (A > B) {
  let X = A;
  A = B;
  B = X;
}

console.log(`Os valores sao: ${A} e ${B}.`);

console.log("");
process.stdout.write('Tecle <Enter> para encerrar... ');
rls.question();

File: PYTHON/Cap04/c04ex02.py
# c04ex02.py

a = float(input("Entre o 1o. valor numerico: "))
b = float(input("Entre o 2o. valor numerico: "))

if (a > b):
  x = a
  a = b
  b = x

print(f"Os valores sao: {a} e {b}.")

print()
input("Tecle <Enter> para encerrar... ")
